x/leverage/keeper/store.go

```diff
@@ -176,9 +173,7 @@ func (k Keeper) getLastInterestTime(ctx sdk.Context) int64 {
 	}
 
 	val := gogotypes.Int64Value{}
-	if err := k.cdc.Unmarshal(bz, &val); err != nil {
-		panic(err)
-	}
+	k.cdc.MustUnmarshal(bz, &val)
```

Comment:
Still needs panic

Reply:
What do you mean? `MustUnmarshal` panics.